src/Admin/AbstractAdmin.php

```diff
@@ -2398,19 +2386,12 @@ private function buildList(): ?FieldDescriptionCollection
         $mapper = new ListMapper($this->getListBuilder(), $this->list, $this);
 
         if (\count($this->getBatchActions()) > 0 && $this->hasRequest() && !$this->getRequest()->isXmlHttpRequest()) {
-            $fieldDescription = $this->createFieldDescription(
-                ListMapper::NAME_BATCH,
-                [
-                    'label' => 'batch',
-                    'sortable' => false,
-                    'virtual_field' => true,
-                ]
-            );
-
-            $fieldDescription->setAdmin($this);
```

Comment:
This call was added in master, I dunno how to reproduce if now I use `add` with a string.
I would say it's not needed since the `add` method works perfectly for every other field.